src/frontend/src/pages/domain/index.tsx

```
@@ -44,6 +52,26 @@ function AdminDataGrid({ domains, pagination }: AdminDataGridProps) {
       headerName: t("admin_maildomains_list.datagrid_headers.updated_at"),
       renderCell: ({ row }: { row: MailDomainAdmin }) => new Date(row.updated_at).toLocaleDateString(i18n.resolvedLanguage),
     },
+    ...(canManageMaildomainAccesses ? [{
+      id: "actions",
+      size: 130,
+      headerName: t("admin_maildomains_list.datagrid_headers.actions"),
+      renderCell: ({ row }: { row: MailDomainAdmin }) => {
+        // eslint-disable-next-line react-hooks/rules-of-hooks
+        const canManageAccesses = useAbility(Abilities.CAN_MANAGE_MAILDOMAIN_ACCESSES, row);
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)